src/main/java/homework/service/impl/FruitTransaction.java

```diff
@@ -0,0 +1,42 @@
+package homework.service.impl;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
```

Comment:
```suggestion
    private Fruit fruit;
```
due to mentor's comments, it is better to make a Fruit class, in case our functionality will expand and fruits will have also price, colors, etc.

Reply:
I did it, but Dima said it was not necessary.